pkg/products/cloudresources/reconciler.go

```diff
@@ -212,7 +217,17 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		}
 	}
 
-	phase, err = r.reconcileCloudResourceStrategies(client)
+	maintenanceDay, _, err := addon.GetStringParameter(ctx, client, operatorNamespace, MaintenanceDay)
+	if err != nil {
+		return phase, err
+	}
+
+	maintenanceHour, _, err := addon.GetStringParameter(ctx, client, operatorNamespace, MaintenanceHour)
+	if err != nil {
+		return phase, err
+	}
+
+	phase, err = r.reconcileCloudResourceStrategies(client, maintenanceDay, maintenanceHour)
```

Comment:
I think this would be applied to AWS cluster also, do we want this or should this be only applied to GCP clusters ? 🤔

Reply:
I think we only want it for GCP clusters. As far as I know maintenance date is set in another way for AWS - these defaults were set prior to this change and will be used when no params are provided. Certainly adding the drop down in OCM is being explicitly asked for just for GCP. https://issues.redhat.com/browse/MGDAPI-5301 
Hope I understand correctly.